README.md

```diff
@@ -121,24 +121,6 @@ After export, they will also be stored in Azure Blob Service under a container w
 
 Alternatively, AKS Periscope can be deployed directly with `kubectl`. See instructions in [Appendix].
 
-
-# Programming Guide

```

Comment:
💡 Suggestion: instead of completely removing can we please add more generic instruction for newbie users please.

* Key aspect for this article if a playing ground to get folks who like to develop on this some basic instructions to:
    * How to build locally.
    * How to test their local changes.

Might be we could simplify to much basic instructions.

Reply:
In my opinion the Dockerfile (and all code in general) is the best documentation. Documentation is dangerous as it may become a pain to maintain.
I'm not anti-documentation but I believe we don't want to explain the very basic usage of `go build` and `go run` I think ^.^

To help test local changes, I agree we could do better by adding some tests :)